killercoda/install.sh

```diff
@@ -20,6 +20,11 @@ function xk6-disruptor() {
 
   mv "$XK6_DISRUPTOR_INSTALL_PATH/xk6-disruptor-linux-amd64" "$XK6_DISRUPTOR_INSTALL_PATH/xk6-disruptor"
 
+  echo "Symlinking K3s kubeconfig to the default path"
```

Comment:
I think adding a symlink is too intrusive and can backfire later due to conflicts with other tools. 
Moreover, I don't think we need this. Exporting the environment variable should be sufficient.

Reply:
I think that if an environment variable, which can be easily overridden works, I don't see the justification for introducing this symlink. Besides, I've never seen other tools doing this.

Reply:
It seems to be a rancher thing, normally tools expect a kubeconfig file in `~/.kube/config`. The `kubectl` binary provided by k3s seems to be custom-made to look for the kubeconfig file on a different place: https://github.com/k3s-io/k3s/blob/10fb39ae60abfc60592497903ba734c7e9a17bdf/pkg/kubectl/main.go#LL27C32-L27C32

Reply:
Still, I don't understand why following this approach that seems not to have any advantage over simply exporting the environment variable as you were doing initially. I only suggested doing this once in the installation instead of doing it on every step to avoid that message to the user.

Reply:
There are no advantages other than keeping the output cleaner. Let's go for the env var for now: https://github.com/grafana/xk6-disruptor-demo/pull/9/commits/49d5a5bf4eb368f7c41803a7d785998cf2236462